dashboarddoc.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: dashboard
+permalink: /dashboarddoc/
+---
+
+User Dashboard Interaction
+==========================
+{: .anchor}
+
+This document illustrates how a Solar Forecast Arbiter user may use the
+Dashboard to complete the following actions:
+
+-   [Creating a new Site](#creating-a-site).
+
+-   [Creating a new Observation or Forecast](#creating-an-observation-or-forecast).
+
+-   [Uploading Data to an Observation or Forecast](#uploading-data).
+
+When a user is filling out a form, a tooltip describing the information
+a user should enter is shown when the field’s name or options are not
+descriptive enough alone. An example of these tooltips is shown below.
+
+<img class="my-3" src="/images/TooltipDetail.png" class="figure"/>
+
+Creating a Site
+---------------
+{: .anchor}
+
+1. Navigate to sites listing page using ‘Sites’ link in the left sidebar. At the top of the Site listing click ‘Create new Site’.
+<img class="my-3" src="/images/SitesListing.png"/>
+
+2. Enter the metadata for your Site. Selecting a site type of 'Power Plant' will prompt you for additional fields as shown below. Sites that are power plants will require  After submission, you will be redirected to a Site page which displays the new Site’s metadata. The Site page will also display links to pages listing its associated Observations and Forecasts. See [Creating an Observation or Forecast](#creating-an-observation-or-forecast).
+- Weather Station
+<img class="my-3" src="/images/SiteForm.png"/>
+- Power Plant
+<img class="my-3" src="/images/plant_site_form.png"/>
+
+
+Creating an Observation or Forecast
+-----------------------------------
+{: .anchor}
+
+To create an Observation or Forecast, an associated site must already exist. See [Creating a Site](#creating-a-site) for instructions.
+
+1.  Navigate to the Site listing page using the ‘Sites’ link in the left sidebar. Select the site for which you are adding an Observation or Forecast.
+
+2.  You may use the “Create new Observation” button on the Site page or on the Site’s Observations listing page.
+- Site Page<br/>
+  <i>Weather Station Site</i>
+  <img class="my-3" src="/images/weather_site.png"/>
```

Comment:
Perhaps we could put one of these images in the section above to show what the created Site page looks like. Then keep the other one here for the reference to the new Obs/Forecast buttons. 

Might make the most sense to move Power Plant Site up to the previous section. Then the two Ashland OR pages are right next to each other in this list.